### ISSUE TITLE
reason

### ISSUE THREAD

**Issue by mvhulten** [sriharshaarangi/BatteryChargeLimit#61]
The README could include a description why someone wants to use this program, i.e. why you want your device to stop charging at a certain level.

**Comment by milux** [comment]
Good point. The README.md should be filled with the relevant information. Will do as soon as there's some time for that.
